crowbar_framework/app/models/api/node.rb

```diff
@@ -150,6 +150,18 @@ def upgrade
       true
     end
 
+    # Remove "pre-upgrade" attribute from given node
+    # We must do it from the node where pacemaker is running
+    def remove_pre_upgrade_role_for(name)
```

Comment:
It's not removing the attribute, it's setting it to `false`. That's the correct approach, but we should name the method correctly.  Also, it's an attribute not a role.  So I suggest `disable_pre_upgrade_attribute`.  The comment needs changing too.